spec/features/dams_object_spec.rb

```diff
@@ -1452,18 +1452,23 @@
     visit dams_object_path @objOtherRight.pid
     expect(page).to_not have_link('', href:"/object/#{@objOtherRight.id}/_1.mp4/download")
   end
+
+  scenario 'user should see an embed link' do
+    visit dams_object_path @obj.pid
+    expect(page).to have_css("a", :text => "Embed")
+  end
```

Comment:
Is it possible to add a test for the embed UI?

Reply:
@lsitu I am already testing for the control. Are you referring to the modal here? As in, test for the control and the modal? Thanks!

Reply:
@dt-ucsd I mean a test for the embed video/audio UI through url`embed/:id`. At lease the video view can be loaded, and it'll be great if the object id and filename of the video/audio link can be verified.

Reply:
Test added.